libs/sdk-ui-filters/src/AttributeFilter/Components/DropdownButton/AttributeFilterDropdownButton.tsx

```diff
@@ -12,28 +12,94 @@ export const ALIGN_POINT = [
 ];
 
 /**
- * @alpha
+ * This is interface AttributeFilter dropdown button.
+ * It is opening Attribute filter dropdown and display title or subtitle, selection details and attribute filter statuses like loading or filtering.
+ *
+ * @remarks
+ * Note for rendering error status see {@link IAttributeFilterErrorProps}
+ * @beta
  */
 export interface IAttributeFilterDropdownButtonProps {
+    /**
+     * Title of attribute {@link  @gooddata/sdk-model#IAttributeFilter} and its related display form {@link  @gooddata/sdk-model#IAttributeDisplayFormMetadataObject}
+     *
+     * @beta
+     */
     title?: string;
+
+    /**
+     * List of selected elements titles separated by coma
+     *
+     * @beta
+     */
     subtitle?: string;
+
+    /**
+     * Selected items count
+     *
+     * @remarks
+     * -  If value is 0 for {@link @gooddata/sdk-model#IPositiveAttributeFilter} means NONE items are selected
+     *
+     * -  If value is 0 for {@link @gooddata/sdk-model#INegativeAttributeFilter} means ALL items are selected
+     *
+     * @beta
+     */
     selectedItemsCount?: number;
 
+    /**
+     * This prop define open or close state of AttributeFilter dropdown
+     *
+     * @beta
+     */
     isOpen?: boolean;
 
+    /**
+     * This prop mean that AttributeFilter is initializing Attribute elements and its internal data
```

Comment:
This prop meanS (repeated several times here).

Reply:
fixed